models/desired_lrp.go

```diff
@@ -13,6 +14,8 @@ import (
 const PreloadedRootFSScheme = "preloaded"
 const PreloadedOCIRootFSScheme = "preloaded+layer"
 
+const maxAllowedSize = 1 * 1024 * 1024 // 1MB in bytes
```

Comment:
Can you rename this constant so that it is clear that it is only about volume mounted files and not the desired_lrp object?